cloudify_nsx/cfy_nsx_security_policy.py

```diff
@@ -42,7 +42,7 @@ def create(**kwargs):
             ctx.logger.info("Used existed %s" % resource_id)
         elif resource_id:
             raise cfy_exc.NonRecoverableError(
-                "We already have such security group"
+                "We already have such security policy"
```

Comment:
Also minor issue, but should be: We already have such a security policy

(or perhaps: Security policy {name} already exists)

The same applies for the security tag below